plugins/funds.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+""" This plugin gives you a nicer overview of the funds that you own.
+
+Instead of calling listfunds and adding all outputs and channels
+this plugin does that for you. 
+
+Author: Rene Pickhardt (https://ln.rene-pickhardt.de)
+
+"""
+
+import json
+import sys
+
+from lightning.lightning import LightningRpc
+from os.path import expanduser
+
+path = expanduser("~/.lightning/lightning-rpc")
+
+rpc_interface = LightningRpc(path)
+
+
+def json_get_funds(request, unit="s"):
+    if unit.lower() == "bitcoin" or unit.lower() == "btc":
```

Comment:
How about creating a dict that maps the long form to the short form? If you do the reverse step, you can also create the inverse map with a simple dict comprehension :-)

Reply:
Yeah dict will be more python style

Reply:
Thx